Source/Core/Core/HW/EXI/EXI_DeviceEthernet.h

```diff
@@ -269,7 +269,7 @@ class CEXIETHERNET : public IEXIDevice
 
     u8 revision_id = 0;  // 0xf0
     u8 interrupt_mask = 0;
-    u8 interrupt = 0;
+    std::atomic<u8> interrupt = 0;
```

Comment:
Would a comment be helpful on why this is atomic when nothing else is?  Or do you think you can gather than from reading references in code?

Without good hand-offs in the design, thread safety can be difficult to see.  Though I don't have any suggestions on how to make things more obvious.

Reply:
Idunno, I can say `// Read and written from multiple threads` but I think that's already implied from being atomic?

Reply:
Yeah fair.  I guess I was thinking about `// Written from Core thread and also a worker thread` but that's not really any more helpful (a code reference would help but it can become outdated).